src/commands/env.rs

```diff
@@ -44,6 +45,15 @@ fn make_symlink(config: &FnmConfig) -> std::path::PathBuf {
     temp_dir
 }
 
+fn dedup_and_merge_path_variable(path: &Path) -> String {
+    let current_path = std::env::var_os("PATH").expect("Can't read PATH env var");
+    let mut split_paths: Vec<_> = std::env::split_paths(&current_path).collect();
+    split_paths.dedup_by(|a, b| a == b || a.to_str().unwrap().contains("fnm_multishells"));
+    split_paths.insert(0, path.to_path_buf());
+    let new_path = std::env::join_paths(split_paths).expect("Can't join paths");
+    new_path.to_str().expect("Can't read PATH").to_owned()
+}
```

Comment:
can we avoid `expect` by returning a `Result`? We are using `thiserror` so we can have nice error reports instead of panics

Reply:
I have added 2 more error reports. The error message is based on the previous code. But one more thing is should I handle `to_str() or just use `unwarp()`? What do you think?
```rust
pub enum Error {
    ...
    #[error("Can't read PATH env var")]
    CantReadPath,
    #[error("Can't join paths: {}", source)]
    CantJoinPaths { source: std::env::JoinPathsError },
}
```